app/presenters/users/index.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+
+module Users
+  class Index < ApplicationPresenter
+    include TableizedView
+
+    def content
+      mustache[:content] ||= begin
+        {
+          title: page_content_heading,
+          resources: users_content
+        }
+      end
+    end
+
+    def page_content_heading
+      t(:title)
+    end
+
+    def event_options
+      EDM::Event.all.map do |event|
+        [event.name, event.id]
+      end
+    end
+
+    protected
+
+    def t(*args, **options)
+      super(*args, options.reverse_merge(scope: 'contribute.pages.users.index'))
+    end
+
+    def users_content
+      {
+        create: {
+          url: new_user_path,
+          text: t('new', scope: 'contribute.users.actions')
+        },
+        table: {
+          has_row_selectors: false, # until we make use of the buttons
+          head_data: users_table_head_data,
+          row_data: users_table_row_data
+        }
+      }
+    end
+
+    def users_table_head_data
+      [
+        t('table.headings.email'),
+        t('table.headings.role'),
+        t('table.headings.events'),
+        t('table.headings.last_sign_in_at'),
+        t('delete', scope: 'contribute.actions')
+      ]
+    end
+
+    def users_table_row_data
+      @users.map do |user|
+        {
+          id: user.id,
+          url: edit_user_path(user),
+          cells: user_table_row_data_cells(user)
+        }
+      end
+    end
+
+    def user_table_row_data_cells(user)
+      [
+        table_cell(user.email),
+        table_cell(user.role),
+        table_cell(user.events.map(&:name).join('; ').truncate(50)),
```

Comment:
This is not very helpful. I suggest either showing the full names of all, or omitting the column, or perhaps just showing the tick or cross.

Reply:
I'd say let's show the complete list then and see if that causes issues in terms of spacing.
It's quite useful to see what events someone is assigned to without having to open the edit page I think.